in_toto/models/layout.py

```diff
@@ -245,45 +253,51 @@ def read(data):
   def _validate_type(self):
     """Private method to ensure that the type field is set to step."""
     if self._type != "step":
-      raise FormatError("Invalid _type value for step (Should be 'step')")
+      raise securesystemslib.exceptions.FormatError(
+          "Invalid _type value for step (Should be 'step')")
 
   def _validate_threshold(self):
     """Private method to check that the threshold field is set to an int."""
     try:
       # int(self.threshold)
       # FIXME: we don't have support for threshold yet
       pass
-    except:
-      raise FormatError("Invalid threshold value for this step")
+    except Exception:
+      raise securesystemslib.exceptions.FormatError(
```

Comment:
Debugging becomes difficult when you catch `Exception` and raise your own without logging or printing the original exception.